File: src/main/java/guru/xgm/vjass/psi/tree/VjassIToken.java
package guru.xgm.vjass.psi.tree;

import com.intellij.psi.tree.IElementType;
import guru.xgm.vjass.lang.VjassLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class VjassIToken extends IElementType {

    public VjassIToken(@NotNull @NonNls String debugName) {
        super(debugName, VjassLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "VjassIToken." + super.toString();
    }
}

File: src/main/kotlin/guru/xgm/language/vjass/psi/tree/VjassIToken.kt
package guru.xgm.language.vjass.psi.tree

import com.intellij.psi.tree.IElementType
import guru.xgm.language.vjass.lang.VjassLanguage
import org.jetbrains.annotations.NonNls

class VjassIToken(debugName: @NonNls String) : IElementType(debugName, VjassLanguage.instance) {
    override fun toString(): String = "VjassIToken." + super.toString()
}
